crates/astria-composer/src/executor/mod.rs

```diff
@@ -130,13 +136,12 @@ impl Executor {
     pub(super) fn new(
         sequencer_url: &str,
         private_key: &SecretString,
-        serialized_rollup_transactions_rx: mpsc::Receiver<SequenceAction>,
```

Comment:
I agree with the change, but not in this PR. Keep passing in the channel here for now, we will construct the `Handle` in a followup